lib/buildmodel.js

```diff
@@ -135,6 +135,66 @@ const BuildModel = class {
     stream(config, response) {
         return this.executor.stream(config, response);
     }
+
+    /**
+     * Get a build based on id
+     * @param  {Object}   config            Config object
+     * @param  {String}   config.table      The table name
```

Comment:
No, I was thinking bout removing it too and construct the config object here instead of inside the plugin.